prime-router/src/main/kotlin/azure/ApiKeysFunctions.kt

```diff
@@ -174,36 +184,34 @@ SwIDAQAB
 @SecuritySchemes(
     value = [
         SecurityScheme(
-            name = "primeSecurity",
+            name = "primeSecurityOAUTH",
             type = SecuritySchemeType.OAUTH2,
             flows = OAuthFlows(
                 authorizationCode = OAuthFlow(
-                    authorizationUrl = "https://hhs-prime.okta.com/oauth/authorize",
-                    tokenUrl = "https://hhs-prime.okta.com/oauth/token",
+                    authorizationUrl = "https://hhs-prime.oktapreview.com/oauth2/default/v1/authorize",
+                    tokenUrl = "https://hhs-prime.oktapreview.com/oauth2/default/v1/token",
                     scopes = [
                         OAuthScope(
-                            name = "org_admin",
-                            description = "Grants write access to single org"
-                        ),
-                        OAuthScope(name = "prime_admin", description = "Grants access to admin operations"),
-                        OAuthScope(name = "user", description = "Grants read access")
+                            name = "openid",
+                            description = "OpenID Request scope"
+                        )
                     ]
                 )
             ),
             description = "OAUTH2 Authorization for Report Stream API Access."
         ),
-        SecurityScheme(
-            name = "primeSecurityAPIKey",
-            type = SecuritySchemeType.APIKEY,
-            paramName = "x-functions-key",
-            description = "Azure Function Key Authorization for Report Stream API Access."
-        ),
         SecurityScheme(
             name = "primeSecurityServerToServer",
             type = SecuritySchemeType.HTTP,
             scheme = "Bearer",
             bearerFormat = "JWT",
             description = "HTTP Bearer Token Authorization for Report Stream API Access."
+        ),
+        SecurityScheme(
+            name = "primeSecurityAPIKey",
```

Comment:
When I tested this locally, I did not see "x-functions-key" in the header of swagger's HTTP request? Can you confirm this works?

Reply:
I did not try it either, thought to try it out on Azure when got a chance, however, good catch, the x-functions-key - it could be in header or qparam, let me check if it's in qparam, if so, need to change the authteniticate(...) a bit

Reply:
> When I tested this locally, I did not see "x-functions-key" in the header of swagger's HTTP request? Can you confirm this works?

Forgot to indicate where the value is: there is an "In" attribute needs to show up in the request, should add it with "header" value..., will fix

Reply:
Now, it has the x-functions-key in header:
x-functions-key -> fake-api-key-00001

Reply:
the authenticate check on each API methods is left as is, so on local you do APIKEY auth or not, it does not matter - it fall back to the default - grant a claim.

On staging (assume CORS policies etc. are properly set), the authenticate will behave as is.

And if APIKEY auth is selected, the swagger UI API docs should work as expected - assuming Azure API management security policies are properly set - this will be verified when swagger ui got deployed  on staging